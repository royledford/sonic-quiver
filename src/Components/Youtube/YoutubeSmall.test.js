import React from 'react'

import YoutubeSmall from './YoutubeSmall'
import renderer from 'react-test-renderer'

it('YoutubeSmall renders', () => {
  const props = {
    id: 'test',
    title: 'test',
    description: 'description',
    thumbnailUrl: 'a/url',
  }
  const tree = renderer.create(<YoutubeSmall {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
