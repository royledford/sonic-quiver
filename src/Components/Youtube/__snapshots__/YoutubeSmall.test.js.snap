// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YoutubeSmall renders 1`] = `
<div
  className="youtubesmall-wrap"
>
  <div
    className="youtubesmall-text"
  >
    <h4
      className="youtubesmall-title"
    >
      test
    </h4>
    <p
      className="youtubesmall-description"
    >
      description
    </p>
  </div>
  <div
    className="youtubesmall-vid-wrap"
    onClick={[Function]}
  >
    <div
      className="youtubesmall-vid"
      style={
        Object {
          "backgroundImage": "url(\\"a/url\\")",
        }
      }
    />
    <div
      className="youtubesmall-cover"
    />
    <svg
      className="youtubesmall-play"
      fill="currentColor"
      height="1em"
      preserveAspectRatio="xMidYMid meet"
      style={
        Object {
          "color": undefined,
          "verticalAlign": "middle",
        }
      }
      viewBox="0 0 40 40"
      width="1em"
    >
      <g>
        <path
          d="m20.1 2.9q4.7 0 8.6 2.3t6.3 6.2 2.3 8.6-2.3 8.6-6.3 6.2-8.6 2.3-8.6-2.3-6.2-6.2-2.3-8.6 2.3-8.6 6.2-6.2 8.6-2.3z m8.6 18.3q0.7-0.4 0.7-1.2t-0.7-1.2l-12.1-7.2q-0.7-0.4-1.5 0-0.7 0.4-0.7 1.3v14.2q0 0.9 0.7 1.3 0.4 0.2 0.8 0.2 0.3 0 0.7-0.2z"
        />
      </g>
    </svg>
  </div>
</div>
`;
